.eslintignore

```diff
@@ -1,3 +1,4 @@
 /index.js
 /index.js.map
 flow-typed
+lib
```

Comment:
👍 

May want to make this `/lib` just because `lib` is a pretty common name. We could also remove `flow-typed`.

Reply:
oh nice catch